examples/aggregate_merging/src/aggregates.rs

```diff
@@ -11,8 +11,8 @@ use esrs::store::{EventStore, SqliteStore, StoreEvent};
 use crate::projectors::CounterProjector;
 use crate::structs::{CommandA, CommandB, CounterError, EventA, EventB, ProjectorEvent};
 
-const A_EVENTS: &str = "a_events";
-const B_EVENTS: &str = "b_events";
+const A_EVENTS: &str = "A";
```

Comment:
Why upcase?

Reply:
I don't think it matters, [e.g.](https://github.com/primait/event_sourcing.rs/blob/f7112d2fdbac13129bd335a04450bba074a82826/examples/postgres_payments/src/bank_account/aggregate.rs#L42)